COLLADAMaya/include/COLLADAMayaSyntax.h

```diff
@@ -25,6 +25,8 @@ namespace COLLADAMaya
     // COLLADA NetAllied Systems
     static const char* TRANSLATOR_VENDOR                            = "NetAllied Systems GmbH";
     static const char* TRANSLATOR_VERSION                           = "1.3.0";
```

Comment:
We won't maintain a version per subproject, but a version of the openCOLLADA overall project
There should be a MAJOR and MINOR version in COLLADABUVersionInfo.h, in addition to revision.
The MAJOR will be set to 1
The MINOR will be set to 6

TRANSLATOR_VERSION should be removed in favor of using version numbers from COLLADABU